### PR TITLE
Allow to link operations on non UTC servers.

### LINKED ISSUE #206: Bill/operation match not done
From last update (or previous one) : 0.4.10/11, the match between operations and bills it not done anymore. 
Here is my date setting on my configuration, because from previous analysis, it is due to problems in managing the UTC/GMT differences: 
``
cozy-term@cozy:~$ date

Tue Mar  8 14:09:01 CET 2016
``
From previous analysis, the involved pieces of code where these one: 
- https://github.com/cozy-labs/konnectors/blob/master/server/lib/link_bank_operation.coffee#L84
If the entry was saved in "local time" ie not UTC, not entry will be retreived
- https://github.com/cozy-labs/konnectors/blob/master/server/lib/save_data_and_file.coffee#L43
The entry.date should be normalized :
``` coffeescript
 entry.date = "#{entry.date.format('YYYY-MM-DD')}T00:00:00.000Z"
```

### PR DESCRIPTION
**Pull request by ZeHiro**
Fix update of operation link.
Fixes #206 
@frankrousseau , works on my installation, can you test?

### CODE REVIEW

File: server/lib/naming.coffee [hunk 1]
```diff
@@ -26,7 +26,9 @@ module.exports =
     #
     getEntryFileName: (entry, options) ->
         name = ""
-
+        # We reset the date modified in save_data_and_file
+        if entry.date?
+            entry.date = new Date(entry.date)
```
Comment (frankrousseau) [inline server/lib/naming.coffee]: Are you sure that a proper date is built from this line of code?

File: server/lib/save_data_and_file.coffee [hunk 2]
```diff
@@ -107,11 +113,11 @@ checkForMissingFiles = (options, callback) ->
 
                         # Then update links it from the current model to
                         # the file.
-                        #date = entry.date.toISOString()
-                        date = moment(entry.date).format('YYYY-MM-DD')
-                        date += "T00:00:00.000Z"
+                        date = moment(new Date entry.date).format 'YYYY-MM-DD'
```
Comment (frankrousseau) [inline server/lib/save_data_and_file.coffee]: I recommend using:

```
entry.date.substring 10
```

File: server/lib/save_data_and_file.coffee [hunk 3]
```diff
@@ -107,11 +113,11 @@ checkForMissingFiles = (options, callback) ->
...
+                        date += 'T00:00:00.000Z'
 
                         model.request 'byDate', key: date, (err, entries) ->
+                            console.log entries.length
```
Comment (frankrousseau) [inline server/lib/save_data_and_file.coffee]: Can you remove this log?

**Comment by frankrousseau** [comment]
Thank you for the fix. I wrote several comments. Can you handle it before merging?

File: server/lib/save_data_and_file.coffee (hunk 2 above)
Comment (ZeHiro) [inline server/lib/save_data_and_file.coffee]: OK

File: server/lib/save_data_and_file.coffee (hunk 3 above)
Comment (ZeHiro) [inline server/lib/save_data_and_file.coffee]: OK

File: server/lib/naming.coffee (hunk 1 above)
Comment (ZeHiro) [inline server/lib/naming.coffee]: yes

**Pull request merged**